Clean radium apps (#24)
STORM-IRIT/Radium-Apps: Sandbox/README.md

# Radium Sandbox

This application has several purpose:
 - It demonstrates standard graphical tools offered by the Radium-Engine: gizmos, picking, skeletal animation, keyframing, renderering, ...
 - It offers a graphical front-end to load and use plugins easily. Just add your plugin to the Plugin path (see graphical user interface), its interface will appear in the command panel and you will be able to use it.

> Internally, we use this application for plugin development and testing of the Radium-Engine library.

Reviewer: Well, this is only partly true (think about painty or Mara)...
```suggestion
Internally, we use this application as an integration and testing application of the Radium-Engine libraries.
```
